startup.sh

```diff
@@ -34,6 +34,15 @@ if [ -d "$ADDITIONAL_LIBS_DIR" ]; then
     cp $ADDITIONAL_LIBS_DIR/*.jar $CATALINA_HOME/webapps/$APP_PATH_PREFIX"geoserver/WEB-INF/lib/"
 fi
 
+# No RCE Log4J Jar (see http://geoserver.org/announcements/2021/12/13/logj4-rce-statement.html)
+if [ "$USE_NORCE_LOG4J_JAR" == 1 ]; then
+  echo "Using the patched norce log4j 1.2.17 jar";
+  # remove malicious log4j-1.2.17.jar
+  rm $CATALINA_HOME/webapps/$APP_PATH_PREFIX"geoserver/WEB-INF/lib/log4j-1.2.17.jar"
```

Comment:
is there an additional `/` after `$APP_PATH_PREFIX` ?

Reply:
Nope, that's correct. See https://github.com/meggsimum/geoserver-docker/pull/2 for details.